src/styles/main.scss

```diff
@@ -1,3 +1,60 @@
-body {
+@import "./utils/variables";
+@import "./utils/mixins";
+@import "./utils/functions";
+
+@import "./normalize.scss";
+
+.page {
+  font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.content {
+  @include centeredContent();
   margin: 0;
 }
+
+.calendar {
+  @include centeredContent();
+  align-content: center;
+  flex-wrap: wrap;
+  padding: $paddings;
+  gap: $gap;
+  width: calcCalendarWidth($gap, $paddings, $daySize);
+  height: calcCalendarHeight($gap, $paddings, $daySize);
+
+  @for $index from 28 through $monthLength {
+    &--month-length_#{$index} {
+      .calendar__day:nth-child(n + #{$index + 1}) {
```

Comment:
use ampersand here

Reply:
Don't really get what do you mean
Inside curly brackets of &--month-length_#{$index}, & means: .calendar--month-length_#{$index}
I can only create variable inside .calendar with it's name and use it here